tests/sqllogictests/suites/stage/copy_into_txn.sql

```diff
@@ -0,0 +1,103 @@
+#################################################################################################################
+# test avoiding duplicates when copy stage files into a table
+#################################################################################################################
+
+statement ok
+create or replace database test_txn_copy;
+
+statement ok
+use test_txn_copy;
+
+statement ok
+create table t1(c int);
+
+onlyif mysql
+statement ok
+begin;
+
+statement ok
```

Comment:
copy has output，here we can use `query`  instead of `statement` to make the test more clear.